content/python/concepts/built-in-functions/terms/all/all.md

```diff
@@ -129,3 +129,72 @@ print(all(my_tuple))
 ```
 
 This is `True` because all items in the tuple are `True`.
+
+## Codebyte Examples
+
+The following example demonstrate the use of `all()` function.
+
+**Checking if all elements in a list satisfy a condition:**
+
+```codebyte/python
+numbers = [2, 4, 6, 8, 10]
+all_even = all(num % 2 == 0 for num in numbers)
+print(all_even)
+
+names = ['Alice', 'Bob', 'Charlie']
+all_long_names = all(len(name) > 3 for name in names)
+print(all_long_names)
+```
+
+The output for the above code is:
```

Comment:
The codebyte is runnable and displays the output, so we don't need to include an output block.